src/BeanstalkClient.php

```diff
@@ -5,37 +5,34 @@
 use Amp\Beanstalk\Stats\Job;
 use Amp\Beanstalk\Stats\System;
 use Amp\Beanstalk\Stats\Tube;
-use function Amp\call;
-use Amp\Deferred;
-use Amp\Promise;
+use Amp\DeferredFuture;
 use Amp\Uri\Uri;
 use Symfony\Component\Yaml\Yaml;
 use Throwable;
 
-class BeanstalkClient {
-    /** @var Deferred[] */
-    private $deferreds;
+class BeanstalkClient
+{
+    /** @var DeferredFuture[] */
+    private array $deferreds;
 
-    /** @var Connection */
-    private $connection;
+    private Connection $connection;
 
-    /** @var string */
-    private $tube;
+    private ?string $tube;
```

Comment:
Do we need `= null` here?

Reply:
I don't know. This is your library, after all. I was hoping I would not need to work on it any more and you could just commit directly to this branch if you want to make changes. I'm not going to protest any change you want to make so just go ahead.